### DISCUSSION TITLE
Covert from Mp3 to M4B

### DISCUSSION THREAD

**Discussion by Puddl36lum** [openaudible/openaudible#791] [Q&A] [answered]
when i originally started using openaudible, i set preferences to convert to .Mp3 format.  is there a way to convert audiobooks from Mp3 format to M4B without downloading the AAX files again?

**Comment by openaudible** [comment, marked as answer]
Not using OpenAudible. Might be possible with ffmpeg, but we recommend downloading again to maintain the best quality. Converting to m4b will be quick.
